tests/sentry/snuba/test_tasks.py

```diff
@@ -158,27 +158,27 @@ def test_no_subscription_id(self):
 
 class BuildSnubaFilterTest(TestCase):
     def test_simple_events(self):
-        snuba_filter = build_snuba_filter(QueryDatasets.EVENTS, "", "count_unique(user)", None,)
+        snuba_filter = build_snuba_filter(QueryDatasets.EVENTS, "", "count_unique(user)", None)
```

Comment:
Do we need to run black again?

Or are you running a diff version of black dan 😬 ?

Reply:
This should be fixed now.